src/taxes/views/TaxCountriesList.tsx

```diff
@@ -152,7 +155,13 @@ export const CountriesList: React.FC<CountriesListProps> = ({ id, params }) => {
       {shop?.countries && (
         <TaxCountryDialog
           open={params?.action === "add-country"}
-          countries={shop?.countries}
+          countries={shop?.countries.filter(
```

Comment:
Try to avoid more complex operations within jsx, create small function for that and use it